### PR TITLE
minor updates and runbook structure

### PR DESCRIPTION
**Pull request by tom-webber**
- update to latest template-documentation-site version 
- add build-and-test workflow
- update workflow actions
- add CNAME record to prevent cert failures
- add runbooks, joining-our-team, and ways-of-working folders

### CODE REVIEW

File: config/tech-docs.yml [hunk 1]
```diff
@@ -19,7 +20,7 @@ enable_search: true
 ga_tracking_id:
 
 # Enable collapsible navigation in the sidebar
-collapsible_nav: false
+collapsible_nav: true
```
Comment (MatMoore) [inline config/tech-docs.yml:23]: Possibly we also need `multipage_nav: true` here

Some context for the GDS template - originally it was developed as a single page, and the nav would be generated from that, but at some point there was functionality added to pick up a multi page structure

Based on the existence of subdirectories, I'm assuming that's what we want, but I haven't got it working locally yet

There is another example here that might be useful https://github.com/alphagov/tech-docs-gem/blob/main/example/config/tech-docs.yml

File: config/tech-docs.yml [hunk 2]
```diff
@@ -30,5 +31,9 @@ max_toc_heading_level: 3
 prevent_indexing: true
 
 show_contribution_banner: true
-github_repo: ministryofjustice/template-documentation-site
+github_repo: ministryofjustice/data-catalogue-runbooks
 github_branch: main
+
```
Comment (MatMoore) [inline config/tech-docs.yml:36]:
```suggestion

default_owner_slack: "#data-catalogue"
owner_slack_workspace: moj.enterprise
```

Haven't tested this - but there is a way to indicate slack links in the review banner

File: source/documentation/joining-our-team/index.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## General Information
+
+- Our team have a stand up each morning at 10:45.
+- We plan our sprints using GitHub Projects.
+- We have our team Retrospective every 2 weeks at the end of each sprint.
```
Comment (murdo-moj) [inline source/documentation/joining-our-team/index.html.md.erb:14]:
```suggestion
- We have our team Retrospective every 2 weeks at the end of each sprint.
- We have a backlog refinement session halfway through every sprint.
```

File: config/tech-docs.yml [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
 
 # Host to use for canonical URL generation (without trailing slash)
-host: https://ministryofjustice.github.io/template-documentation-site
+host: https://runbooks.data-catalogue.service.justice.gov.uk
 
 # Header-related options
 show_govuk_logo: true
-service_name: Template Documentation Site
+service_name: Data Catalogue Runbooks
 service_link: /
```
Comment (MatMoore) [inline config/tech-docs.yml:9]:
```suggestion
service_link: /
phase: Alpha
```

File: source/documentation/joining-our-team/index.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+## Tools
+
+### Communication
+
+Our main chat platform is Slack, and this is our preferred method of communication.
+
+We use Slack huddles in the [#data-catalogue channel](https://moj.enterprise.slack.com/archives/C03QZ776JVA)
+for pairing or direct calls.
+
+Developers link PR requests in the [#dpl-developers channel](https://moj.enterprise.slack.com/archives/C05AABSKFLJ)
```
Comment (murdo-moj) [inline source/documentation/joining-our-team/index.html.md.erb:25]:
```suggestion
Developers link PRs in the [#dpl-developers channel](https://moj.enterprise.slack.com/archives/C05AABSKFLJ)
```

File: config/tech-docs.yml [hunk 6]
```diff
@@ -1,16 +1,17 @@
 # See the available options: https://tdt-documentation.london.cloudapps.digital/configure_project/global_configuration/#configure-your-documentation-site
 
 # Host to use for canonical URL generation (without trailing slash)
-host: https://ministryofjustice.github.io/template-documentation-site
+host: https://runbooks.data-catalogue.service.justice.gov.uk
 
 # Header-related options
 show_govuk_logo: true
```
Comment (MatMoore) [inline config/tech-docs.yml:7]:
```suggestion
show_govuk_logo: false
```

Not sure we meet the criteria for using govuk branding for this
https://github.com/alphagov/tech-docs-template?tab=readme-ov-file#tech-docs-template

> You’re welcome to use the template even if your service isn’t considered part of GOV.UK, but your site or  service must not:
>
> identify itself as being part of GOV.UK
> use the crown or GOV.UK logotype in the header
> use the GDS Transport typeface
> suggest that it’s an official UK government website if it’s not

**Review by MatMoore — APPROVED** [review]
Basic structure looks good to me. Do we want to split out the runbook further or just have as a single page?

File: source/documentation/joining-our-team/index.html.md.erb [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+- questions or discussions around the service
+- to request someone else from the team's help
+- when we're away for lunch or some other appointment
+- general musings
+
+If you can't make standup, post your standup update here.
+
+### GitHub
+
+Access to Data Catalogue's GitHub repositories and projects is managed in Terraform.
```
Comment (murdo-moj) [inline source/documentation/joining-our-team/index.html.md.erb:49]:
```suggestion
Access to Data Catalogue's GitHub repositories and projects is managed in Terraform.
Here are some repos we use as a team which you should familarise yourself with:
- https://github.com/ministryofjustice/find-moj-data
- https://github.com/ministryofjustice/data-platform

We use MoJ's AWS hosting services for new services if we can, which are run by service teams:
- https://user-guide.cloud-platform.service.justice.gov.uk/
- https://user-guide.modernisation-platform.service.justice.gov.uk/
```

Comment (tom-webber) [reply source/documentation/joining-our-team/index.html.md.erb:49]: We can throw this into the ways of working in a separate PR - [see AP's WoW](https://technical-documentation.data-platform.service.justice.gov.uk/documentation/team/ways-of-working.html)

**Comment by tom-webber** [comment]
> Basic structure looks good to me. Do we want to split out the runbook further or just have as a single page?

Runbooks can be added as separate pages, I like [AP's runbook style](https://github.com/ministryofjustice/analytical-platform-runbooks/tree/main/source/documentation/runbooks)

File: config/tech-docs.yml (hunk 1 above)
Comment (tom-webber) [reply config/tech-docs.yml:24]: Great suggestion. I've added this, and it _looks_ to be working locally

**Review by MatMoore — APPROVED** [review]

**Pull request merged**